uber/config.py

```diff
@@ -394,6 +419,18 @@ def _is_intstr(s):
 c.HOURS = ['{:02}'.format(i) for i in range(24)]
 c.MINUTES = ['{:02}'.format(i) for i in range(60)]
 
+c.DAYS_OF_WEEK = {'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'}
+if c.ONE_DAYS_ENABLED and c.PRESELL_ONE_DAYS:
+    _day = c.EPOCH.date()
+    while _day <= c.ESCHATON.date():
+        _name = _day.strftime('%A')
+        _val = c.create_enum_val(_name)
+        c.BADGES[_val] = _name
+        c.BADGE_OPTS.append((_val, _name))
+        c.BADGE_VARS.append(_name.upper())
+        c.BADGE_RANGES[_val] = c.BADGE_RANGES[c.ONE_DAY_BADGE]
+        _day += timedelta(days=1)
```

Comment:
Yeah, the reason why I ended up doing it this way is that the original approach basically required an event which wanted to presell single day passes to add a set of configuration options VERY PRECISELY.  As in, if they added an option like ``friday = "Friday Pass"`` then everything would have broken, because the option MUST be ``friday = "Friday"`` due to how we're doing the lookups.

That kind of restriction is just begging for weird, hard-to-diagnose bugs, which is why I went this route.